From e1c26acfb0bf9e610ac97fba90269f9c3a148dc2 Mon Sep 17 00:00:00 2001
From: Timur Sultanov <SultanovTS@yandex.ru>
Date: Sun, 6 Feb 2022 18:24:30 +0300
Subject: [PATCH libXrandr] Disable malloc(0) check

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d3fa7f6..788f51d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -45,7 +45,7 @@ m4_ifndef([XORG_MACROS_VERSION],
           [m4_fatal([must install xorg-macros 1.8 or later before running autoconf/autogen])])
 XORG_MACROS_VERSION(1.8)
 XORG_DEFAULT_OPTIONS
-XORG_CHECK_MALLOC_ZERO
+# XORG_CHECK_MALLOC_ZERO
 
 # Check randr configuration, strip extra digits from package version to
 # find the required protocol version
-- 
2.25.1

